requirements/edx/doc.txt

```diff
@@ -905,7 +905,7 @@ openedx-django-require==2.1.0
     # via -r requirements/edx/base.txt
 openedx-django-wiki==2.0.0
     # via -r requirements/edx/base.txt
-openedx-events==8.3.0
+openedx-events @ git+https://github.com/open-craft/openedx-events@navin/configurable-handler
```

Comment:
**TODO**: update this requirement once https://github.com/openedx/openedx-events/pull/249 is merged